Tests/APITesters/SwiftAPITester/SwiftAPITester/PurchasesAPI.swift

```diff
@@ -166,7 +165,9 @@ private func checkPurchasesSupportAPI(purchases: Purchases) {
     #endif
 }
 
+@available(*, deprecated) // Ignore deprecation warnings
 private func checkPurchasesSubscriberAttributesAPI(purchases: Purchases) {
+    _ = purchases.attribution
```

Comment:
This needs to check the type:
```suggestion
    let _: Attribution = purchases.attribution
```

And I would move this out of this method since it's not deprecated (and if it were, we wouldn't want to ignore that warning)

Reply:
touche

Reply:
✅